llvm/lib/SYCLLowerIR/ModuleSplitter.cpp

```diff
@@ -1244,7 +1244,11 @@ splitSYCLModule(std::unique_ptr<Module> M, ModuleSplitterSettings Settings) {
 
 bool canBeImportedFunction(const Function &F) {
   if (F.isIntrinsic() || F.getName().starts_with("__") ||
-      !llvm::sycl::utils::isSYCLExternalFunction(&F))
+      !( // SYCL_EXTERNAL property is not recorded for a declaration
```

Comment:
I would put this comment above the `if` instead of in the middle of the condition

Reply:
Comment moved.